feature/manager.go

```diff
@@ -63,6 +63,10 @@ type Config struct {
 	// CustomFeatures is a set of custom features to advertise in each
 	// set.
 	CustomFeatures map[Set][]lnwire.FeatureBit
+
+	// NoPeerStorage, when set to false, enables storage of backup data
```

Comment:
suggest: `NoPeerStorage unsets any bits signalling support for peer storage` or something along those lines .